src/lib/components/ConfirmTransactions.js

```diff
@@ -1,10 +1,12 @@
 import React, { useState } from "react";
 import Modal from "react-bootstrap/Modal";
-import { Markdown } from "./Markdown";
-import { displayGas, displayNear, Loading } from "../data/utils";
-import { useNear } from "../data/near";
+import { formatNearAmount } from "near-api-js/lib/utils/format";
```

Comment:
I don't like the implementation of the `formatNearAmount`. It doesn't understand yoctoNear. Let's stick with displayNear for consistency. We can fix it later